ui/imports/shared/popups/send/SendModal.qml

```diff
@@ -287,18 +290,26 @@ StatusDialog {
                         visible: d.isSelectedHoldingValidAsset && !d.isERC721Transfer
                         AmountToSend {
                             id: amountToSendInput
-                            Layout.fillWidth:true
+
+                            Layout.fillWidth: true
                             isBridgeTx: d.isBridgeTx
                             interactive: popup.interactive
                             selectedSymbol: d.selectedSymbol
                             maxInputBalance: d.maxInputBalance
                             currentCurrency: popup.currencyStore.currentCurrency
+
```

Comment:
will the multiplier by default be 1 or 0 ?

Reply:
`multiplierIndex` default value should be 0 bc it means 10^0, effectively multiplier is 1